cookbooks/aws-parallelcluster-slurm/recipes/config/config_head_node.rb

```diff
@@ -282,6 +283,21 @@
   retry_delay 2
 end unless redhat_on_docker?
 
+template "#{node['cluster']['scripts_dir']}/slurm/check_login_nodes_stopped.sh" do
+  source 'slurm/head_node/check_login_nodes_stopped.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0700'
+  variables(
+    cluster_name: node['cluster']['cluster_name'] || node['cluster']['stack_name'],
```

Comment:
I'm not a big fan of this `||` logic. Are there any circumstances where one of these two variables does not exist? Do we have one of them that exists for certain?

Reply:
I am not sure. I use this because other place that uses cluster_name use `||` logic like this.

Like https://github.com/hehe7318/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-environment/recipes/config/custom_actions_setup.rb line 24